.github/workflows/publish-registry.yml

```diff
@@ -50,9 +48,12 @@ jobs:
           mkdir -p dist
           cp src/registry.json dist/registry.json
           cp src/signature.json dist/signature.json
-      - name: Deploy registry
-        uses: peaceiris/actions-gh-pages@de7ea6f8efb354206b205ef54722213d99067935
+      - name: configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
         with:
-          personal_token: ${{ secrets.METAMASKBOT_TOKEN }}
-          publish_dir: ./dist
-          destination_dir: latest
+          role-to-assume: ${{ vars.AWS_ROLE_ARN }}
+          role-session-name: ghactionssession
+          aws-region: ${{ vars.AWS_REGION }}
+      - name: Deploy registry
+        run: |
+          aws s3 cp ./dist s3://${{ vars.AWS_BUCKET_NAME }}/latest --recursive --acl private
```

Comment:
Can we create a CloudFront invalidation as well?

Reply:
Yes, that's something to be added, thanks for raising it up!